src/ensembl/src/content/app/help/Help.tsx

```diff
@@ -0,0 +1,90 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { useLocation } from 'react-router';
+
+import useApiService from 'src/shared/hooks/useApiService';
+
+import HelpMenu from './help-menu/HelpMenu';
+import {
+  IndexArticle,
+  TextArticle,
+  RelatedArticles,
+  HelpArticleGrid,
+  VideoArticle
+} from 'src/shared/components/help-article';
+
+import { Menu as MenuType } from 'src/shared/types/help-and-docs/menu';
+import {
+  TextArticle as TextArticleType,
+  VideoArticle as VideoArticleType,
+  IndexArticle as IndexArticleType
```

Comment:
Would it be better to just append `Type` to the original definition to avoid ambiguity?

Reply:
Updated type names, and also fixed a little bug with the Help Menu that I noticed during today's code review (in certain scenarios, the menu required two clicks to open).